src/Events/Span.php

```diff
@@ -101,11 +101,11 @@ public function start(float $startTime = null) // TODO separate out timing from
     /**
      * Stop the Timer
      *
-     * @param integer|null $duration
+     * @param float|null $duration
      *
      * @return void
      */
-    public function stop(int $duration = null) // TODO separate out timing from Event
```

Comment:
What is the purpose for the change from integer to float here? The span duration is given in milliseconds, and I believe that is an integer value. We aren't getting fractional milliseconds. Is there a situation you've found that is different?

Reply:
Tbh, i have no clue actually.. seems I just following the changes on the last issue. Probably stand on the integer still be the best option for declaration. No Issue for that :)